pkg/apis/core/validation/validation_test.go

```diff
@@ -14050,25 +14050,38 @@ func TestValidateServiceCreate(t *testing.T) {
 		tweakSvc: func(s *core.Service) {
 			s.Spec.ExternalIPs = []string{"127.0.0.1"}
 		},
-		numErrs: 1,
+		numErrs: 2,
```

Comment:
I assume this is the additional "required" error?  We should adjust the test loop to set this properly, so each case is really testing just one thing

Reply:
The new "required" was externalTrafficPolicy is required when ExternalIP is set for ClusterIP service.
I have adjusted the new tests to test one thing only.